Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
```

Comment:
The `cmds` under `dev_server` are quite lengthy and contain many parameters. Refactoring this command into a shell script or function can improve readability and maintainability, allowing for easier modifications in the future without digging through a long line.